app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+class InvalidCommandError(Exception):
+    ...
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) != 3 or command_list[0] != "mv":
+        raise InvalidCommandError("Unknown command.")
+
```

Comment:
add `command_name, origin_path, new_path = command_list` and use for more readable code.